controllers/namespacescope_controller.go

```diff
@@ -98,6 +98,27 @@ func (r *NamespaceScopeReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 		return ctrl.Result{}, nil
 	}
 
+	nssObjectList := &operatorv1.NamespaceScopeList{}
+	if err := r.Client.List(ctx, nssObjectList); err != nil {
+		return ctrl.Result{}, err
+	}
+
+	licenseAccepted := false
+
+	for _, nss := range nssObjectList.Items {
+		if nss.GetDeletionTimestamp() != nil {
+			continue
+		}
+		if nss.Spec.License.Accept {
+			licenseAccepted = true
+			break
+		}
+	}
+
+	if !licenseAccepted {
+		klog.Info("Accept license by changing .spec.license.accept to true in the NamespaceScope CR.")
```

Comment:
```suggestion
		klog.Infof("Accept license by changing .spec.license.accept to true in the NamespaceScope: %s", req.NamespacedName)
```

I wonder if we would like to add an event to NamespaceScope CR. Something like [this](https://github.com/IBM/ibm-common-service-operator/blob/5755e3dcb53fec75c6a151b5a434d7e68adb2e62/controllers/commonservice_controller.go#L306)
```
r.Recorder.Event(instance, corev1.EventTypeWarning, "LicneseAcceptance", fmt.Sprint("Accept license by changing .spec.license.accept to true in the NamespaceScope CR."))
```